public/javascripts/cohortController.js

```diff
@@ -14,3 +14,12 @@ var cohortController = function($scope,  $http, $state) {
     });
   };
 };
+
+var cohortStudentsController = function($scope, $http, $state) {
+
+  $scope.showStudent = function(student) {
+    $scope.$parent.$parent.currentStudent = student;
```

Comment:
Hm. Interesting. I think it's not worth worrying about since they both work?

On Fri, Apr 29, 2016 at 2:08 PM Nora Mohamed <notifications@github.com>
wrote:

> In public/javascripts/cohortController.js
> <https://github.com/davidabrahams/Spring-Initiative/pull/161#discussion_r61621357>
> :
>
> > @@ -14,3 +14,12 @@ var cohortController = function($scope,  $http, $state) {
> >      });
> >    };
> >  };
> > +
> > +var cohortStudentsController = function($scope, $http, $state) {
> > +
> > +  $scope.showStudent = function(student) {
> > +    $scope.$parent.$parent.currentStudent = student;
>
> that's what i'd assume, but in the view student data controller I'm just
> using $scope.currentStudent and it works. when i did the parent thing,
> the controller wouldn't change your current student to be the newly clicked
> one
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/davidabrahams/Spring-Initiative/pull/161/files/4564c5995d55d41f88d42c58d73ffcdc14eb369d#r61621357>
>